sktime/collections/classification/bakeoff_2017/bakeoff_2017_collection.py

```diff
@@ -0,0 +1,127 @@
+"""TSC Bakeoff 2017 Collection."""
+
+from sktime.collections.base import BaseCollection
+
+
+class TSCBakeOff2017(BaseCollection):
+    """Collection of datasets, estimators, and metrics from TSC Bake Off 2017.
+
+    Examples
+    --------
+    >>> from sktime.collections import TSCBakeOff2017
+    >>> collection = TSCBakeOff2017()
+    >>> available_categories = collection.available_categories()
+    >>> all_items = collection.get("all")
+    >>> dataset_loaders = collection.get("dataset_loaders")
+    >>> classifiers = collection.get("estimators")
+    """
+
+    _tags = {
+        "authors": "jgyasu",
+        "maintainers": "jgyasu",
+        "collection_type": "mixed",
+        "info:name": "The Great Time Series Classification Bake Off",
+        "info:source": "https://arxiv.org/pdf/1602.01711",
+    }
+
+    def _get(self):
+        """Return a dict of items (datasets, estimators, metrics)."""
+        return items
+
+
+dataset_loaders = [
+    "LargeKitchenAppliances",
```

Comment:
is this correct, or should it not be `UCRUEADataset("LargeKitchenAppliances")` etc?